saritasa_invocations/_config.py

```diff
@@ -240,7 +241,12 @@ class K8SDefaultSettings:
     db_config: K8SDBSettings | None = None
     port: str = "443"
     auth: str = "github"
-    pod_label: str = "app.kubernetes.io/component"
+    component_selector: str = "app.kubernetes.io/component"
+    pod_command: str = (
```

Comment:
`pod_command` is too common name, let's use something like `get_single_pod_name_command` or provide ability to configure only `--selector` and leave `kubectl get pods --no-headers --output jsonpath='{{.items[0].metadata.name}}` as hardcoded part